tests/driver_soft_uart/main.c

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
```

Comment:
I wonder if it would make sense to just integrate the test of the soft UART into the periph UART.

Reply:
Unless we make the UART API extendable those will use different function names though.